src/controllers/google.auth.controller.ts

```diff
@@ -0,0 +1,94 @@
+import passport from 'passport';
+import bcrypt from 'bcrypt';
+import { User } from '../models/index';
+import dotenv from 'dotenv';
+import { Request } from 'express';
+
+dotenv.config();
```

Comment:
We are repeating ourselves here, the dotenv should be configured in the main entry file and we should be able to use it anywhere else without writing this line everytime
```suggestion
```